### PR TITLE
Add Naruto OST - Samidare , Kimetsu No Yaiba Links

### PR DESCRIPTION
**Pull request by nishantc7**
Fixes #{ISSUE_NUMBER}

Demo: https://ongaku-${PULL_REQUEST}.surge.sh

#### This PR is for [Anime Name].

- [ ] This issue is for enhancement/bug fixing.
- [x] This issue is an addition to JSON database.
	- [ ] Pull Request Title is of form `Added [Anime Name]`.
- [ ] This addition is already on ongaku.
	- [ ] This PR is an improvement to current addition.
- [ ] I have checked my addition locally and it is working and won't break existing entries.
- [x] I have gone through [Contribution Guidelines](https://github.com/anshumanv/ongaku/blob/master/CONTRIBUTING.md).
- [ ] This pull request will be closed if I failed to update it even once in a week.

### CODE REVIEW

File: js/data.json [hunk 1]
```diff
@@ -1471,5 +1471,25 @@ osts = [
 		"link": "https://www.dropbox.com/s/adhvu2r1646c2j0/Gohan%20ya-san%20Megumi.mp3?dl=1",
 		"name": "Amaama to Inazuma OST - Gohan ya-san Megumi",
 		"img": "https://www.dropbox.com/s/suvtvzcdm5rptxy/amaama%204.jpg?dl=1"
+	},
+	{
+		"link": "https://www.dropbox.com/s/obccd111g0a4ho3/28%20-%20Samidare.mp3?dl=0",
+		"name": "Naruto OST - Samidare",
+		"img": "https://www.dropbox.com/s/c36rmzzdmkvpw54/jiraya_img.jfif?dl=0"
+	},
+	{
```
Comment (anshumanv) [inline js/data.json]: You should move Opening and Ending objects in appropriate arrays, this array is just for OST so leave Naruto here and move others to their respective location.

File: js/data.json [hunk 2]
```diff
@@ -1471,5 +1471,25 @@ osts = [
...
+		"link": "https://www.dropbox.com/s/uo9kqpcw2mz470t/Demon%20SlayerOPGurenge%20by%20LiSA%20HD.mp3?dl=0",
+		"name": "Kimetsu No Yaiba - OP ",
```
Comment (anshumanv) [inline js/data.json]: You should add the name of the Op, same with all songs, like you did with Naruto

`Kimetsu No Yaiba - OP: Gurenge`

**Review by anshumanv — CHANGES_REQUESTED** [review]
Looking great! 😄 Some changes.

Thanks for adding Kimetsu No Yaiba. ❤️

File: js/data.json (hunk 1 above)
Comment (nishantc7) [reply js/data.json]: I understand.

File: js/data.json (hunk 2 above)
Comment (nishantc7) [reply js/data.json]: So can I call the ep 19 ending as kamaboko gonpanchiro no uta? just kidding. I've made the changes as requested.

Comment (anshumanv) [reply js/data.json]: 🤣 🤣 🤣

File: js/data.json [hunk 3]
```diff
@@ -747,6 +752,16 @@ endings = [
 		"link": "https://www.dropbox.com/s/fvutawwuqjfocov/Brian%20the%20Sun%20-%20Maybe.mp3?dl=1",
 		"name": "Amaama to Inazuma ED - Maybe",
 		"img": "https://www.dropbox.com/s/nw2bwf87t8nudh1/111223.jpg?dl=1"
+	},
+	{
+		"link": "https://www.dropbox.com/s/l1q963ejmsf5bi0/Romaji%20Lyrics%5D.mp3?dl=0",
+		"name": "Kimetsu No Yaiba - ED - from the edge",
```
Comment (anshumanv) [inline js/data.json]:
```suggestion
		"name": "Kimetsu No Yaiba - ED: From the edge",
```

File: js/data.json [hunk 4]
```diff
@@ -747,6 +752,16 @@ endings = [
...
+		"img": "https://www.dropbox.com/s/drvo1xj8lrb1ynb/88eb5cc2fcebaeb2bd76987a4ca4ad2b.png?dl=0"
+	},
+	{
+		"link": "https://www.dropbox.com/s/dxc3bosu4h0pv4h/Demon%20Slayer%3A%20Kimetsu%20no%20Yaiba%20ED%20%28Ep19%29%20Full%20-%20Kamado%20Tanjirou%20no%20Uta%20%28Lyrics%29.mp3?dl=0",
+		"name": "Kimetsu No Yaiba - Ep-19 ED - Kamado Tanjirou no Uta",
```
Comment (anshumanv) [inline js/data.json]:
```suggestion
		"name": "Kimetsu No Yaiba - Ep-19 ED: Kamado Tanjirou no Uta",
```

File: js/data.json [hunk 5]
```diff
@@ -1471,5 +1486,10 @@ osts = [
 		"link": "https://www.dropbox.com/s/adhvu2r1646c2j0/Gohan%20ya-san%20Megumi.mp3?dl=1",
 		"name": "Amaama to Inazuma OST - Gohan ya-san Megumi",
 		"img": "https://www.dropbox.com/s/suvtvzcdm5rptxy/amaama%204.jpg?dl=1"
+	},
+	{
+		"link": "https://www.dropbox.com/s/obccd111g0a4ho3/28%20-%20Samidare.mp3?dl=0",
+		"name": "Naruto OST - Samidare",
+		"img": "https://www.dropbox.com/s/c36rmzzdmkvpw54/jiraya_img.jfif?dl=0"
```
Comment (anshumanv) [inline js/data.json]: This image seems invalid, .jfif format?

**Review by anshumanv — COMMENTED** [review]
Almost there!

File: js/data.json (hunk 5 above)
Comment (nishantc7) [reply js/data.json]: :(

Comment (nishantc7) [reply js/data.json]: my bad to have you go through this so many times. I'll make sure I won't repeat myself in future contributions.

**Comment by starkblaze01** [comment]
Hi @nishantc7, great to see new contributor and anime lover!
Can you make a small change in your links?
Your sharing link of dropbox looks something like this:  `https://www.dropbox.com/s/abc/xyz.ext?dl=0`

Notice the `dl=0` at the last?
Can you please change it to `dl=1`, else it won't be a download link. You can try changing it to 1 and you will see that download starts automatically once you change it. You can check other dropbox links in the same file for reference.

**Review by starkblaze01 — CHANGES_REQUESTED** [review]
Songs edition looks great, only these changes needed.
Hope to see you contributing again ;)

File: js/data.json (hunk 5 above)
Comment (anshumanv) [reply js/data.json]: Lol no worries, it's alright 😄

**Review by anshumanv — APPROVED** [review]
Thanks for your contribution. 😄

**Pull request merged**